cmd/api/server.go

```diff
@@ -149,7 +149,7 @@ var serverCMD = &cobra.Command{
 			configFiles.NewConfigFilesK8sRepo(
 				k8sRepo,
 				k8sClient,
-				logrus.WithField("context", "microservice-config-files-repo"),
+				logrus.WithField("context", "microservice-config-files-service"),
```

Comment:
```suggestion
				logrus.WithField("context", "microservice-config-files-repo"),
```

This should still stay as repo as you're passing the logger to the repo.